sandbox.go

```diff
@@ -531,7 +531,15 @@ func (sb *sandbox) SetKey(basePath string) error {
 	}
 
 	for _, ep := range sb.getConnectedEndpoints() {
-		if err = sb.populateNetworkResources(ep); err != nil {
+		ipv6 := false
+		n, err := ep.getNetworkFromStore()
+		if err != nil {
+			log.Warnf("IPv6 enabled check failed for endpoint %s (%s).  Could not retrieve the network: %v",
+				ep.Name(), ep.ID(), err)
+		} else {
+			ipv6 = n.enableIPv6
+		}
+		if err = sb.populateNetworkResources(ep, ipv6); err != nil {
```

Comment:
Yeah I agree. `ipv6Enabled` should really be a property of endpoints. Moreover I think it is really necessary to make it work during live restore. Otherwise I am not sure where that configuration comes from when we restart docker.